src/makeInfinityAdder.js

```diff
@@ -6,7 +6,23 @@
  */
 
 function makeInfinityAdder() {
-  // write code here
+  let res = 0;
+
+  const adder = (value) => {
+    if (value !== undefined) {
+      res += value;
+
+      return adder;
+    } else {
+      const output = res;
+
+      res = 0;
+
+      return output;
+    }
```

Comment:
Else is redundant here
```suggestion
    }
    
    const output = res;

    res = 0;

    return output;
```